apps/web/components/layout/header/mobile-nav-icon/index.tsx

```diff
@@ -0,0 +1,13 @@
+"use client"
+
+import { cn } from "@/lib/utils"
+
+export function MobileNavIcon({ isOpen, onClick }: { isOpen?: boolean, onClick?: (event: React.MouseEvent<HTMLDivElement>) => void }) {
+    return (
+        <div onClick={onClick} className="select-none relative w-7 h-5 mx-auto rotate-0 transition duration-500 ease-in-out pointer">
+            <span className={cn("block absolute h-[3px]  mx-auto bg-white rounded-full opacity-100  transition duration-500 ease-in-out origin-left left-[5px]", isOpen ? "rotate-45 -top-[2px]  w-full" : "rotate-0 top-0 w-2/3")}></span>
```

Comment:
@leandrogavidia -- the className `w-2/3` might be the one causing the distortion. Using relative values on pixel perfect animations may cause distortions on them, unless it is intentional for the design.